src/app/lib/config/services/sinhala.ts

```diff
@@ -66,6 +66,12 @@ export const service: DefaultServiceConfig = {
         audioPlayer: 'හඬ වාදකය',
         videoPlayer: 'වීඩියෝ ධාවකය',
       },
+      liveExperiencePage: {
+        live_coverage: 'සජීවී වාර්තාකරණය ',
+        breaking: 'අලුත්ම පුවතක් ',
+        posted_at: 'යාවත්කාලීන කළේ ',
+        summary: 'සාරාංශය ',
```

Comment:
Just noticed some translations have spaces at the end throughout a handful of files

Reply:
Yea good spot. Looks like they are coming from the translation service that way. I'll check with @JonBeeb to see if its safe enough to trim off the whitespace at the end.